types/hkt/doorSensor/system.schema.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$id": "https://akenza.io/hkt/doorSensor/system.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "System",
+  "type": "object",
+  "properties": {
+    "hwVersion": {
+      "title": "Hardware version",
+      "type": "integer",
+      "description": "Hardware version",
+      "hideFromKpis": true
+    },
+    "swVersion": {
+      "title": "Software version",
+      "type": "integer",
+      "description": "Software version",
+      "hideFromKpis": true
+    },
+    "Working mode": {
+      "title": "Mode",
+      "type": "integer",
+      "description": "Working mode 1: During the counting hours, the device will periodically report the number of times the door sensor is openedand closed in the previous cycle and the total accumulatedtimes. Outside the counting hours, the device runs in trigger mode. Working mode 2: Trigger mode (Default): Every time the tamper alarm is triggered or the door is opened or closed, the devicereportsdata immediately.",
```

Comment:
Working mode 1: During the counting hours, the device will periodically report the number of times the door sensor is opened and closed in the previous cycle and the total accumulated times. Outside the counting hours, the device runs in trigger mode. Working mode 2: Trigger mode (Default): Every time the tamper alarm is triggered or the door is opened or closed, the device reports data immediately.